azure.gemspec

```diff
@@ -50,3 +49,4 @@ Gem::Specification.new do |s|
   s.add_development_dependency('yard',                '~> 0.8')
   s.add_development_dependency('vcr',                 '~> 3.0')
 end
+
```

Comment:
Using that git property is risky. It converts CLRF and LF vice versa freely and do not know the consequences. So, for now, I am Ok with adding an empty line or removing it altogether. (I am Ok with the red marker in PRs).